suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml

```diff
@@ -122,7 +122,8 @@ tests:
               count: 1
               size: 10G
             listener_port: 5001
-            allow_host: "*"
+            hosts:
```

Comment:
Support both "*" and list of nodes. If we continue to use `hosts`, probably check out the other test suites which needs changes.
or go with allow_host to support only '*' with if condition.

Reply:
@sunilkumarn417 please check now per log below

```
2024-08-07 07:37:26,629 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1528 - long running command on 10.0.65.219 -- podman login --username cp --password eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJJQk0gTWFya2V0cGxhY2UiLCJlbnYiOiJzdGFnZSIsImlhdCI6MTcxMzc3NTQ1NSwianRpIjoiYzZmMTc5N2JiOTA4NGQ2YzkwZWUxNjQ3MzNhOWFmNDAifQ.DgpzR7VvRQ3ZUFwD_CCCH57gy3wuv_BRvva7F_sXCjE cp.stg.icr.io with 3600 seconds
2024-08-07 07:37:27,633 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.ceph.py:1553 - b'Login Succeeded!'
2024-08-07 07:37:27,633 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1560 - Command completed on 2024-08-07 07:37:27.633261
2024-08-07 07:37:27,633 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : subsystem add
2024-08-07 07:37:27,634 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 subsystem add  --subsystem nqn.2016-06.io.spdk:cnode1 --max-namespaces 32 on 10.0.65.219 timeout 600
2024-08-07 07:37:27,634 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : subsystem add
2024-08-07 07:37:27,635 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 subsystem add  --subsystem nqn.2016-06.io.spdk:cnode2 --max-namespaces 32 on 10.0.65.219 timeout 600
2024-08-07 07:37:28,865 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:28,865 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding subsystem nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-07 07:37:28,865 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : gw info
2024-08-07 07:37:28,866 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --format json --server-address 10.0.65.219 --server-port 5500 gw info  on 10.0.65.219 timeout 600
2024-08-07 07:37:29,043 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:29,043 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding subsystem nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:29,044 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : gw info
2024-08-07 07:37:29,044 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --format json --server-address 10.0.65.219 --server-port 5500 gw info  on 10.0.65.219 timeout 600
2024-08-07 07:37:29,918 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:29,918 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', '{\n    "cli_version": "1.2.17",\n    "version": "1.2.17",\n    "name": "client.nvmeof.rbd.ceph-nvme-1-wzk17m-node6.gllzxo",\n    "addr": "10.0.65.219",\n    "port": "5500",\n    "bool_status": true,\n    "error_message": "Success",\n    "spdk_version": "24.01.1",\n    "load_balancing_group": 1,\n    "hostname": "ceph-nvme-1-wzk17m-node6",\n    "group": "",\n    "status": 0\n}\n')
2024-08-07 07:37:29,918 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-07 07:37:29,919 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 listener add  --host-name ceph-nvme-1-wzk17m-node6 --traddr 10.0.65.219 --trsvcid 5002 --subsystem nqn.2016-06.io.spdk:cnode2 on 10.0.65.219 timeout 600
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', '{\n    "cli_version": "1.2.17",\n    "version": "1.2.17",\n    "name": "client.nvmeof.rbd.ceph-nvme-1-wzk17m-node6.gllzxo",\n    "addr": "10.0.65.219",\n    "port": "5500",\n    "bool_status": true,\n    "error_message": "Success",\n    "spdk_version": "24.01.1",\n    "load_balancing_group": 1,\n    "hostname": "ceph-nvme-1-wzk17m-node6",\n    "group": "",\n    "status": 0\n}\n')
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 listener add  --host-name ceph-nvme-1-wzk17m-node6 --traddr 10.0.65.219 --trsvcid 5001 --subsystem nqn.2016-06.io.spdk:cnode1 on 10.0.65.219 timeout 600
2024-08-07 07:37:30,935 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:30,935 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode2 listener at 10.0.65.219:5002: Successful\n')
2024-08-07 07:37:30,936 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command yum install -y nvme-cli fio on 10.0.64.123 timeout 600
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode1 listener at 10.0.65.219:5001: Successful\n')
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode1 --host '*' on 10.0.65.219 timeout 600
2024-08-07 07:37:32,134 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:32,135 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Allowing open host access to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:32,135 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command rbd create rbd/9LJ2-image0 --size 10G on 10.0.64.123 timeout 600
2024-08-07 07:37:32,278 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:32,278 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.rbd.rbd_utils.py:107 - Command execution complete
2024-08-07 07:37:32,278 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is 0
2024-08-07 07:37:32,279 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : namespace add
2024-08-07 07:37:32,279 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 namespace add  --subsystem nqn.2016-06.io.spdk:cnode1 --rbd-pool rbd --rbd-image 9LJ2-image0 on 10.0.65.219 timeout 600
2024-08-07 07:37:33,076 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command modprobe nvme-fabrics on 10.0.64.123 timeout 600
2024-08-07 07:37:33,096 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command nvme show-hostnqn on 10.0.64.123 timeout 600
2024-08-07 07:37:33,115 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:33,116 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-07 07:37:33,116 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode2 --host nqn.2014-08.org.nvmexpress:uuid:a92ca03f-7c47-4940-adc2-483468d0326c on 10.0.65.219 timeout 600
2024-08-07 07:37:33,288 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:33,288 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:33,289 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:34,051 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:34,052 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding host nqn.2014-08.org.nvmexpress:uuid:a92ca03f-7c47-4940-adc2-483468d0326c to nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-07 07:37:34,052 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command rbd create rbd/T09U-image0 --size 10G on 10.0.64.123 timeout 600
2024-08-07 07:37:34,165 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.rbd.rbd_utils.py:107 - Command execution complete
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is 0
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : namespace add
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 namespace add  --subsystem nqn.2016-06.io.spdk:cnode2 --rbd-pool rbd --rbd-image T09U-image0 on 10.0.65.219 timeout 600
2024-08-07 07:37:35,158 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:35,158 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-07 07:37:35,158 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode2: Successful\n')
```